setup.cfg

```diff
@@ -50,8 +50,9 @@ install_requires =
     docker
     importlib-metadata; python_version<"3.8"
     jsonschema>=4.4.0
-    protobuf==3.20.1
-    requests<=2.28.1
+    protobuf~=3.20
+    urllib3<2
+    requests
```

Comment:
did we check if this wont create an issue any more ?

Reply:
+1 We need to stick to that version until we test and validate the latest version

Reply:
I did test it.

Reply:
Plus the `requests` has documentation on this.